docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: mesheryctl system status | Meshery
+permalink: reference/mesheryctl/system/status
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
+list: exclude
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl system status
+
+## Description
+
+{% for command_hash in site.data.mesheryctlcommands.lifecycle.system.start.command %}{% assign command = command_hash[1] %}
+{{ command.description }}
+{% endfor %}
+
+<!-- Basic usage of the command -->
+<pre class="codeblock-pre">
+  <div class="codeblock">
+    mesheryctl system status [flags]
+  </div>
+</pre>
+
+## Examples
+
+{% for command_hash in site.data.mesheryctlcommands.lifecycle.system.status.command %}{% assign command = command_hash[1] %}
+{{ command.description }}
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  {{ command.usage }}
+  </div>
+</pre>
+{% endfor %}
+{% for flag_hash in site.data.mesheryctlcommands.lifecycle.system.status.flag %}{% assign flag = flag_hash[1] %}
+{{ flag.description }}
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  {{ flag.usage }}
+  </div>
+</pre>
+{% endfor %}
+<br/>
+
+
+<!-- Options/Flags available in this command -->
+## Options
+
+<pre class="codeblock-pre">
+  <div class="codeblock">
+    {% for flag_hash in site.data.mesheryctlcommands.lifecycle.system.status.flag %}{% assign flag = flag_hash[1] %}
+    {{ flag.flag }} # {{ flag.description }}
+    {% endfor %}
+  </div>
+</pre>
+<br/>
```

Comment:
![image](https://user-images.githubusercontent.com/59508546/120266670-077bdf00-c2c0-11eb-9069-e1fa55fb39e5.png)

Can we include `--help` here? Or can we leave it blank? @leecalcote

Reply:
@alphaX86 many of the options have blank fields and --help is a universal flag. 
IMO we can include this here, but @leecalcote what would you think about this?

Reply:
I think that we should be using {% include %} to incorporate such information, but to avoid repeating ourselves in the code of the docs.

Reply:
So, can we leave a blank for now then? We can focus this on a new PR regarding this new change. IMHO, there are numerous changes needed in docs now

Reply:
Great suggestion, @alphaX86 👍